app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -1078,7 +1078,7 @@ public void onFailure(Call<SusiResponse> call, Throwable t) {
 
     private void updateDatabase(final long prevId, final String message, final boolean isDate, final String date, final String timeStamp , final boolean mine,
```

Comment:
@chiragw15 don't you think it's a good idea to add javadocs for the methods we are using to communicate with the server?